src/service.js

```diff
@@ -161,20 +162,28 @@ async function startService ({ peerId, port, peerAnnounceAddr, awsClient, connec
 
     // TODO move to networking
     service.connectionManager.addEventListener('peer:connect', connection => {
+      cancelsPerPeer.set(
+        connection.detail.remotePeer.toString(),
+        new LRU({ max: 200 })
+      )
+
       try {
         telemetry.increaseCount('bitswap-connections')
         telemetry.increaseGauge('bitswap-active-connections')
       } catch (err) {
-        logger.warn({ err, remotePeer: connection.remotePeer }, 'Error while peer connecting')
+        logger.warn({ err, remotePeer: connection.detail.remotePeer.toString() }, 'Error while peer connecting')
```

Comment:
This is likely broken since libp2p update, where now we use `addEventListener` instead of directly `.on`